lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,9 +1,14 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Card from './Card';
 
 const Cards = props => {
   return (
     <div className="cards-container">
+      {props.cards.map( (card, index) => {
+        return <Card  key={index} card={card} />
```

Comment:
So this is a tricky project to make a reasonable key for because Lambda didn't include keys in the dummy data but for your edification the reason this is not ideal is that your indices can change.  If you started messing around with the order of these cards with CRUD operations you might start getting non-unique keys which would be bad.